Signal/test/util/UtilTest.m

```diff
@@ -35,7 +35,6 @@ - (NSString *)filterUnsafeFilenameCharacters;
 @implementation UtilTest
 
 -(void) testRemoveAllCharactersIn {
-    testThrows([@"" removeAllCharactersIn:nil]);
```

Comment:
This is not a nullable param, and we assert it's presence which was causing the test to crash.